src/_posts/languages/php/2000-01-01-start.md

```diff
@@ -510,37 +510,43 @@ one or more configuration files:
 
 ## Example: Configuring Rate Limiting
 
-The following process setup rate limiting, 1 requests per second per IP.
+In the following example we will set a rate limit of `1 request per second per IP`.
 
-1. Create a file 'nginx-http.conf'
+Create a `nginx-http.conf` file at the root of your project:
 
-```
+```nginx
 limit_req_zone $binary_remote_addr zone=mylimit:10m rate=1r/s;
 ```
 
-2. Create a file 'nginx-app.conf'
+Then create a `nginx-app.conf` file at the root of your project:
 
-```
+```nginx
 location /login {
     limit_req zone=mylimit;
     try_files $uri $uri/ /index.php?$query_string;
 }
 ```
 
-3. Modify your composer.json
+Then modify your `composer.json` to add `nginx-http-includes` and `nginx-includes` config:
 
-```
+```json
 {
-  [...]
   "extra": {
     "paas": {
-      "nginx-http-includes": ["nginx-http.conf"]
+      "nginx-http-includes": ["nginx-http.conf"],
       "nginx-includes": ["nginx-app.conf"]
     }
   }
 }
 ```
 
+{% warning %}
+  This configuration will not work with `location /` as it can't be overrided in the PHP buildpack.
+  If you want to apply a rate limit on all endpoints of your application you need to set-up 2 apps
+  (1 web app and 1 reverse proxy app) by using our
```

Comment:
```suggestion
  (one web app and one reverse proxy app) by using our
```
Nitpick: The standard form is to write smaller numbers up to ten as words, with numerals saved for larger numbers.